### PR TITLE
Announcing flowfuse

### PR DESCRIPTION
**Pull request by iskerrett**
Blog post to announce FlowFuse

### CODE REVIEW

**Review by ZJvandeWeg — APPROVED** [review]

File: src/blog/2023/08/flowforge-is-now-flowfuse.md [hunk 1]
```diff
@@ -1,11 +1,10 @@
 ---
-title: FlowForge is now FlowFuse: New identity but same vision for industrial data integration
-
+title: "FlowForge is now FlowFuse New identity but same vision for industrial data integration"
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/flowforge-is-now-flowfuse.md:2]:
```suggestion
title: "FlowForge is now FlowFuse: New identity but same vision for industrial data integration"
```

File: src/blog/2023/08/flowforge-is-now-flowfuse.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    - news
+---
+
+We are happy to announce that FlowForge is changing its name to FlowFuse. Changing our corporate identity wasn’t our top priority but a recent trademark challenge has promoted us to create a new brand for our company. 
+
+<!--more-->
+
+We believe that this new name better reflects our core mission and aspirations. Just as electricity fuses elements to create energy, the FlowFuse platform fuses together data, ideas, processes, and technologies to generate a powerful force of innovation and transformation.
+
+Selecting a new name for a company or product is never easy. Some of the requirements we set for a new name were: 1) it should be reasonably close to FlowForge so it will be easier to transition the brand identity, 2) it should even better reflect our mission, and 3) keeping the FF acronym will help with environment variable prefixes :-). I think we have accomplished all the above and am delighted to move forward as FlowFuse.
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/flowforge-is-now-flowfuse.md:20]: I wonder if we should mention I proposed FlowRed, just a joke?

Comment (iskerrett) [reply src/blog/2023/08/flowforge-is-now-flowfuse.md:20]: I wouldn't. Some people might not get it and some might be offended.

File: src/blog/2023/08/flowforge-is-now-flowfuse.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+**What to Expect Next**
+
+Our team is heads down building a platform to allow organizations to reliably deliver Node-RED applications in a continuous, collaborative, and secure manner. We continue to deliver releases of new features and innovation on a regular four week release cadence. Our commitment to our customers is FlowFuse will continue to deliver on this promise.
```
Comment (knolleary) [inline src/blog/2023/08/flowforge-is-now-flowfuse.md:24]:
```suggestion
Our team is focussed on building a platform to allow organizations to reliably deliver Node-RED applications in a continuous, collaborative, and secure manner. We'll continue to release regular updates packed with new features and enhancements that keeps moving us towards this goal.
```

Comment (ZJvandeWeg) [reply src/blog/2023/08/flowforge-is-now-flowfuse.md:24]:
```suggestion
Our team is heads down building a platform to allow organizations to reliably deliver Node-RED applications in a continuous, collaborative, and secure manner. We'll continue to release regular updates packed with new features and enhancements that keeps moving us towards this goal.
```

File: src/blog/2023/08/flowforge-is-now-flowfuse.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+title: "FlowForge is now FlowFuse: New identity but same vision for industrial data integration"
+subtitle: New identity but same vision for industrial data integration
```
Comment (iskerrett) [inline src/blog/2023/08/flowforge-is-now-flowfuse.md:3]: We are repeating the subtitle in the main title and the subtitle so it looks odd. I would suggest we remove 'New identity but same vision for industrial data integration' from the title.

File: src/blog/2023/08/flowforge-is-now-flowfuse.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+---
+title: "FlowForge is now FlowFuse: New identity but same vision for industrial data integration"
```
Comment (knolleary) [inline src/blog/2023/08/flowforge-is-now-flowfuse.md:2]:
```suggestion
title: "FlowForge is now FlowFuse"
```

As per @iskerrett suggestion

**Pull request merged**